src/quo2/components/list_items/messages_home_item.cljs

```diff
@@ -0,0 +1,136 @@
+(ns quo2.components.list-items.messages-home-item
+  (:require [clojure.string :as string]
+            [re-frame.core :as re-frame]
+            [status-im.utils.handlers :refer [<sub]]
+            [status-im.utils.datetime :as time]
+            [quo2.foundations.typography :as typography]
+            [quo2.components.notifications.info-count :refer [info-count]]
+            [quo2.components.icon :as icons]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.avatars.user-avatar :as user-avatar]
+            [quo.react-native :as rn]
+            [status-im.ui.screens.chat.sheets :as sheets]
+            [quo.platform :as platform]))
+
+(def max-subheader-length 100)
+
+(defn truncate-literal [literal]
```

Comment:
I know this function was copied from the old implementation, but I'll just comment it'd be preferable to use the core functions `count` instead of `length` and `subs` instead of `substring`.